content/configuration/source.md

```diff
@@ -13,9 +13,68 @@ The source settings contains:
 
 Sources are bound to specific channels in the communications configuration. To learn more, read the [Source and Executor Bindings](/configuration/communication/#source-and-executor-bindings) section.
 
+## General merging strategy
```

Comment:
Consider current ToC:
  * [General merging strategy](#general-merging-strategy)
    * [Example](#example)
  * [Recommendations](#recommendations)
    * [Merging Strategy](#merging-strategy)
  * [Syntax](#syntax)


IMO it would be better to have structure like:

  * [Kubernetes resource events](#kubernetes-resource-events)
    * [Merging strategy](#merging-strategy)
  * [Recommendations](#recommendations)
    * [Merging Strategy](#merging-strategy)
  * [Syntax](#syntax)

``````markdown
## Kubernetes resource events

A source essentially notifies a channel about events for configured resources filtered by specified namespaces.

<!-- maybe here we can add a syntax snippet? e.g.


```yaml
sources:
  'k8s-events':
    kubernetes:
      resources:
        - name: v1/services
          namespaces:
            include:
              - ".*"
          events:
            - create
            - delete
            - error
```

or just remove the syntax snippet from `## Recommendations` section and have it just in one place under `## Syntax`. 

Both approaches are fine by me.
-->

### Merging strategy

When a channel binds to more than one source, the notifications are merged across all sources.

Let's say you have a resource defined in more than one source but wired with different events and namespaces.

```yaml
sources:
  'k8s-events':
....
``````
---
the "general" term also doesn't fit currently. We have two things:
- `kubernetes.resources`
- `kubernetes.recommendations`

both have some approach to “merging”, so I don't see that one can be called general and the second alternative.